tests/rptest/tests/redpanda_oauth_test.py

```diff
@@ -7,47 +7,58 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0
 
-import time
-import functools
-import json
-
+from ducktape.tests.test import Test
+from rptest.clients.rpk import RpkTool
 from rptest.clients.python_librdkafka import PythonLibrdkafka
-from rptest.services.redpanda import SecurityConfig, make_redpanda_service
+from rptest.services.redpanda import LoggingConfig, SecurityConfig, make_redpanda_service
 from rptest.services.keycloak import KeycloakService
 from rptest.services.cluster import cluster
-
-from ducktape.tests.test import Test
-from rptest.services.redpanda import make_redpanda_service
-from rptest.clients.rpk import RpkTool
 from rptest.util import expect_exception
-
 from confluent_kafka import KafkaException
 
 CLIENT_ID = 'myapp'
+TOKEN_AUDIENCE = 'account'
 EXAMPLE_TOPIC = 'foo'
 
+log_config = LoggingConfig('info',
+                           logger_levels={
+                               'security': 'trace',
+                               'pandaproxy': 'trace',
+                               'kafka/client': 'trace',
+                               'http': 'trace'
+                           })
+
```

Comment:
Intentional trace level logs or left over from testing?

Reply:
> Intentional trace level logs or left over from testing?

Trace logging for the interesting components is intentional. Successful runs strip excessive logging back to info.

Reply:
👍